### PR TITLE
Update remaining references to -ngx

### LINKED ISSUE #52: [Other] Update name to paperless-ngx in docs
https://github.com/paperless-ngx/paperless-ngx/issues/4#issuecomment-1039890021

> I would go through the places where the project name is mentioned and add the x to it. Only actually code (variable names) could remain unchanged for backwards compatibility.

### PR DESCRIPTION
**Pull request by qcasey**
This might be sufficient to close #52. Can anyone else drum up more lines to change?

### CODE REVIEW

File: Dockerfile
Comment (stumpylog) [inline Dockerfile:130]: While this line is being updated, maybe it should change to `LABEL org.opencontainers.image.authors=`?  It's what replaced the [previous MAINTAINER instruction](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated) anyway.

There's also [other `LABEL`s which could be set](https://github.com/opencontainers/image-spec/blob/main/annotations.md#pre-defined-annotation-keys) like vendor or version, for some future work.

Comment (qcasey) [reply Dockerfile]: Great point, I added a few of the static ones.

**Comment by shamoon** [comment]
@schnuffle will take care of ci.yml and then I think thats all I can find, i.e. LGTM™️

File: src/documents/templates/registration/login.html
Comment (qcasey) [inline src/documents/templates/registration/login.html:15]: I think this change will mess up Crowdin

Comment (shamoon) [reply src/documents/templates/registration/login.html:15]: Not really, it'll change this item to be a different item that needs translation but thats fine, it should be that way. And then I think it'll take the translation from elsewhere since its the same string

Comment (shamoon) [reply src/documents/templates/registration/login.html:15]: Ah, youre right tho will need to regenerate the strings file but thats easy, 1 sec. Let me make sure its not crazy

Comment (qcasey) [reply src/documents/templates/registration/login.html:15]: Okay I'll defer to you on that

Comment (shamoon) [reply src/documents/templates/registration/login.html:15]: Oh wait duh thats Django template, same idea but let me look at that

Comment (shamoon) [reply src/documents/templates/registration/login.html:15]: OK famous last words here but I think this should be fine, updated `django.po` and thats the "source" for all others, should flow via Crowdin when this is merged into `dev`

*should* 😅

**Comment by schnuffle** [comment]
Okay reverted the merge that somehow got into my commit but kept my ci.yml changes.

**Comment by schnuffle** [comment]
And finally moved my changes to feature-ci-ngx-dirs

File: src/documents/templates/registration/login.html
Comment (shamoon) [reply src/documents/templates/registration/login.html:15]: Looks like these changes got lost maybe in the Schnuffle stuff. I’ll sort it out in a bit

Comment (qcasey) [reply src/documents/templates/registration/login.html:15]: > Famous last words here

Thanks :smile:

**Review by shamoon — APPROVED** [review]
Ok, LFG (pardon my French)

**Comment by shamoon** [comment]
And now we await Crowdin 🤞

**Pull request merged**
